x-pack/platform/plugins/shared/streams_app/public/components/stream_detail_features/stream_description/use_stream_description_api.ts

```diff
@@ -49,13 +49,29 @@ export const useStreamDescriptionApi = ({
   const save = useCallback(
     async (nextDescription: string) => {
       setIsUpdating(true);
+
+      let stream;
+      if (Streams.GroupStream.Definition.is(definition.stream)) {
+        stream = omit(definition.stream, ['name', 'updated_at']);
+      } else {
+        stream = {
+          ...omit(definition.stream, ['name', 'updated_at']),
+          ingest: {
+            ...definition.stream.ingest,
+            processing: {
+              ...omit(definition.stream.ingest.processing, ['updated_at']),
+            },
+          },
+        };
+      }
```

Comment:
This was left out in https://github.com/elastic/kibana/pull/242904 cc @AlejandroFrndz